rudder-web/src/main/resources/load-page/rudder-not-loaded.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en"><head>
+
+
+
+
+<meta content="text/html; charset=UTF-8" http-equiv="Content-Type">
+<title>Rudder - Load Page</title>
+<link media="screen" type="text/css" rel="stylesheet" href="/files/rudder-login.css">
+
+<link type="image/png" rel="icon" href="/images/favicon.png">
+<link type="image/x-icon" rel="shortcut icon" href="/images/favicon.ico">
+
+<style type="text/css">
+li { list-style-type: disc  }
+</style>
+
+</head><body>
+<div id="login">
+  <div class="logo"><h1><img alt="Rudder" src="/files/logoRudder.jpg" height="111" width="511"></h1></div>
+
+    <div class="logo">
+      <h2>404 Rudder had a problem while loading you page</h2>
+      <div style="margin-top: 20px;color: #fff;font-size: 1.2em">
+        This may be caused by:
+
+        <ul style=" width: 300px; margin: 10px 30%; text-align: left; ">
+          <li>The URL you are trying to access doesn't exist.</li>
+          <li>A component such as LDAP or PostgreSQL is missing.</li>
+          <li>Jetty couldn't load the Rudder application.</li>
+        </ul>
+      More information can be found in the Rudder webapp log file.
```

Comment:
Looks good, but i'd replace this by:
```
Error 404: Page not found

This may be caused by:
* A non-existing URL
* An essential component (LDAP or PostgreSQL) that is not accessible
* The web application beeing unable to load

More information can be found in the Rudder webapp log file in /var/rudder/webapp/ on the Rudder server.
```